app/admin/admin.rb

```diff
@@ -24,10 +40,14 @@
 
   form do |f|
     f.inputs "User Details" do
+      f.semantic_errors *f.object.errors.keys
```

Comment:
This contains a pretty good explanation of the "splat" operator: http://stackoverflow.com/questions/918449/what-does-the-unary-operator-do-in-this-ruby-code